app/kafka/manager/ClusterManagerActor.scala

```diff
@@ -185,8 +185,11 @@ class ClusterManagerActor(cmConfig: ClusterManagerActorConfig)
           tds <- eventualDescriptions
           tis = tds.descriptions.map(TopicIdentity.from(bl, _))
         } yield {
+          bl.list.map(_.id.toInt)
+          // check if any nonexistent broker got selected for reassignment
+          require(isValidBrokerList(bl, brokers), "Nonexistent broker(s) selected")
```

Comment:
This will throw an exception, we should add error handling such that this error message is surfaced back through the UI.